src/screens/ChatScreen/SignedChatScreen.tsx

```diff
@@ -25,9 +25,11 @@ const SignedChatScreen = () => {
     goBackFromChatScreen,
     goToChatInfoScreen,
     sendChat,
-    isLoadingFetchAllMessage
+    isLoadingFetchAllMessage,
+    updateChatContinuity
   } = useChatScreenHook(SIGNED);
 
+  const updatedChats = updateChatContinuity(chats);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new function `updateChatContinuity` is being used to transform the `chats` data. However, it's not clear how this function handles errors. If there's an error during the transformation process, it could lead to unexpected behavior or crashes. Consider adding error handling within the `updateChatContinuity` function or here where it's called.
```diff
- const updatedChats = updateChatContinuity(chats);
+ let updatedChats;
+ try {
+   updatedChats = updateChatContinuity(chats);
+ } catch (error) {
+   console.error('Failed to update chat continuity: ', error);
+   // Handle the error appropriately, e.g., show an error message to the user
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->